README.md

```diff
@@ -1,2 +1,4 @@
 # .github
-Welcome to minilabus
+Welcome to minilabus  
+
+Bienvenue sur le Github de minilabus
```

Comment:
Peux-tu traduire 'The Medical Imaging and NeuroInformatic laboratory (MINi lab.) explores ways to increase the clinical relevance of medical imaging. More specifically the usefulness of diffusion magnetic resonance imaging and tractography. The laboratory is located in the Science Faculty of the Université de Sherbrooke. The Université de Sherbrooke is a french-speaking University (undergraduate courses and administrative interactions).' et l'ajouté ici ?

Et ajouté un logo?